test/helper.rb

```diff
@@ -8,19 +8,21 @@
 # Do not add any other code to this code block. Simplecov and
 # coveralls only until the next code block:
 
-require 'simplecov'
-require 'coveralls'
-
-SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
-])
-
-SimpleCov.start do
-  add_filter '/test/'
-  add_group 'Resources', 'lib/resources'
-  add_group 'Matchers', 'lib/matchers'
-  add_group 'Backends', 'lib/inspec/backend'
+  require 'simplecov'
```

Comment:
Indentation starts weird? Did you mean to wrap this in the same CI ENV?

Reply:
Yes its weird right now. I'm having a issue where coverage seems to be running early. If you look at the travis build you'll see it 'send to coveralls' just before it starts executing the tests. If you look at the coveralls link you can see that its only a few files from the repo.

Reply:
Look here: https://travis-ci.org/inspec/inspec/jobs/534128587#L2977

Reply:
Indentation is weird because I've moved the ENV conditional around it a few different ways.